src/Learning/SFA.DAS.Learning/Application/Notification/Handlers/ApprenticeshipWithdrawnCommand.cs

```diff
@@ -0,0 +1,48 @@
+﻿using MediatR;
+using SFA.DAS.Employer.Shared.UI;
+using SFA.DAS.Learning.Application.Notification.Templates;
+
+namespace SFA.DAS.Learning.Application.Notification.Handlers;
+
+public class ApprenticeshipWithdrawnCommand : NotificationCommandBase, IRequest<NotificationResponse>
```

Comment:
Do we need to update this too? I wasn't sure on this one as it does call notifications which is scoped specifically to apprenticeships, but then it is a command which can happen to any "learning" object.